Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "BarricadeKit",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "BarricadeKit",
+            targets: ["BarricadeKit"]),
+    ],
+    dependencies: [
```

Comment:
The empty array of dependencies and the basic comments in the file can be removed.